packages/test-interface-criteria/src/criteria/TestOpportunityBookable.js

```diff
@@ -31,7 +31,7 @@ const TestOpportunityBookable = createCriteria({
       mustNotBeOpenBookingInAdvanceUnavailable,
     ],
     [
-      'Must be within validFromBeforeStartDate window',
+      'Must be within the booking window (`validFromBeforeStartDate` and/or `validThroughBeforeStartDate`) if one exists',
```

Comment:
@nickevansuk btw there is now a mechanism for not having to type out the same message for each criteria. See `mustAllowFullRefundOfferConstraint` for an example (packages/test-interface-criteria/src/criteria/criteriaUtils.js:417)

Reply:
Nice!! Some refactoring on the cards...